src/lib/stitching/exchange/schema.ts

```diff
@@ -0,0 +1,26 @@
+import { createStressLink } from "./link"
+import {
+  makeRemoteExecutableSchema,
+  transformSchema,
+  RenameTypes,
+  RenameRootFields,
+} from "graphql-tools"
+import { readFileSync } from "fs"
+
+export const executableStressSchema = async () => {
+  const stressSDL = readFileSync("src/data/exchange.graphql", "utf8")
+  const stressLink = createStressLink()
+
+  const schema = await makeRemoteExecutableSchema({
+    schema: stressSDL,
+    link: stressLink,
+  })
+
+  // Return the new modified schema
+  return transformSchema(schema, [
+    new RenameTypes(name => {
+      return `Ecommerce${name}`
```

Comment:
Why did we decide to do this? Seems like slightly redundant naming as `ecommerce` orders are our current only understanding of orders-- if we add new concepts in the future seems like we could namespace those accordingly.